drizzle-orm/src/expo-sqlite/query.ts

```diff
@@ -43,7 +44,7 @@ export const useLiveQuery = <T extends Pick<AnySQLiteSelect, '_' | 'then'> | SQL
 		return () => {
 			listener?.remove();
 		};
-	}, []);
+	}, deps);
```

Comment:
wouldn't the query be referencing a new object on each re-render?

Reply:
Here's a simple example demonstrating how adding the `deps` array allows for more granular control over the re-execution of a query.

If you only pass the query, it will change with every re-render of the component. However, by specifying a separate dependency, the query will update only when necessary.

https://playcode.io/1942093

Reply:
Oh yes of course it would be better!
It would avoid wrapping the query in a `useCallback` before passing it to `useLiveQuery`